packages/discovery-provider/src/tasks/index_payment_router.py

```diff
@@ -410,7 +428,8 @@ def attempt_index_recovery_transfer(
             and_(
                 USDCTransactionsHistory.user_bank == receiver_bank_account,
                 USDCTransactionsHistory.method == USDCTransactionMethod.send,
-                USDCTransactionsHistory.tx_metadata == sender_account,
+                # External transfers are indexed on the owner of the token account
```

Comment:
nit: indexed on the owner of the *recipient* token account, right?

Reply:
Potentially confusing comment. The _original_ transfer that we are attempting to undo would have been indexed on the owner account of the recipient. I think that's what you're saying.

Reply:
Updated comment to hopefully be clearer